addon-test-support/@ember/test-helpers/dom/trigger-event.js

```diff
@@ -4,14 +4,27 @@ import settled from '../settled';
 import { nextTickPromise } from '../-utils';
 
 /**
-  Triggers an event on the specified target.
-
-  @public
-  @param {string|Element} target the element or selector to trigger the event on
-  @param {string} eventType the type of event to trigger
-  @param {Object} options additional properties to be set on the event
-  @return {Promise<void>} resolves when the application is settled
-*/
+ * Triggers an event on the specified target.
+ *
+ * @public
+ * @param {string|Element} target the element or selector to trigger the event on
+ * @param {string} eventType the type of event to trigger
+ * @param {Object} options additional properties to be set on the event
+ * @return {Promise<void>} resolves when the application is settled
+ *
+ * @example
+ * <caption>Using triggerEvent to Upload a file
+ * When using triggerEvent to upload a file the `eventType` must be `change` and  you must pass an
+ * array of [Blob](https://developer.mozilla.org/en-US/docs/Web/API/Blob) as `options`.</caption>
+ *
+ * ```javascript
+ * triggerEvent(
+ *   'input.fileUpload',
+ *   'change',
+ *   [new Blob(['Ember Rules!'])]
+ *  );
```

Comment:
a minor indentation issue, this bracket should be moved one char left

Reply:
Thanks @ro0gr. Fixed.